gi/object.cpp

```diff
@@ -2444,6 +2437,85 @@ gjs_object_set_gproperty (GObject      *object,
     g_free (underscore_name);
 }
 
+static JSBool
+gjs_override_property(JSContext *cx,
+                      unsigned   argc,
+                      jsval     *vp)
+{
+    jsval *argv = JS_ARGV(cx, vp);
```

Comment:
I completely missed the existence of `gjs_parse_call_args` this morning, so I will switch to `JS::CallArgs` after all.